How to contribute

```diff
@@ -0,0 +1,30 @@
+This documentation is primarily aimed at Plaid employees who would like to edit the collection. For instructions on using the collection, see the README.
```

Comment:
Probably should give this file an .md extension so that it renders properly in GitHub